ceph/rbd/snapshot.py

```diff
@@ -0,0 +1,74 @@
+from ceph.rbd.rbd import Rbd
+from tests.rbd.exceptions import CreateCloneError, ProtectSnapError, SnapCreateError
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class RbdSnapShot(Rbd):
```

Comment:
Would it make sense to have the object/class name as `Snapshot`

Reply:
done, renamed to Snapshot